.changeset/modern-apples-travel.md

```diff
@@ -0,0 +1,6 @@
+---
+'@chainlink/ea-bootstrap': patch
+'@chainlink/kaiko-adapter': patch
+---
+
+Add API_TIMEOUT to env default overrides, with override for Kaiko
```

Comment:
Feel free to re-phrase, just want to include the actual value in the changeset.

```suggestion
Add API_TIMEOUT to env default overrides, with an override of 30s for Kaiko
```

Reply:
Done!